packages/button-theme/src/Fieldset.tsx

```diff
@@ -0,0 +1,14 @@
+import { applyTheme, StyleConfig } from '@button-inc/component-library/Fieldset';
+import styles from './styles';
+
+const config: StyleConfig = {
+  defaultProps: {
+    size: 'medium',
+  },
+  staticProps: ['fullWidth'],
+  forwardProps: ['size', 'variant', 'disabled', 'required', 'fullWidth'],
```

Comment:
The `fullWidth` version has some ugly spacing, but since we're going to need to do proper styling once we have designs anyway, I think it's okay for now